src/paste-markdown-html.ts

```diff
@@ -123,7 +123,7 @@ function linkify(element: HTMLAnchorElement, label: string): string {
   let markdown = ''
 
   // Don't linkify user mentions like "@octocat"
-  if (isUserMention(element)) {
+  if (isUserMention(element) || isTeamMention(element)) {
```

Comment:
while here, are there other types that shouldn't become urls?

Reply:
@mattcosta7 Not that I know of.